src/YQMenuBar.cc

```diff
@@ -197,6 +197,30 @@ YQMenuBar::setItemEnabled( YMenuItem * item, bool enabled )
 }
 
 
+void
+YQMenuBar::setItemVisible( YMenuItem * item, bool visible )
+{
+    QObject * qObj = static_cast<QObject *>( item->uiItem() );
+
+    if ( qObj )
+    {
+        QMenu * menu = qobject_cast<QMenu *>( qObj );
+
+        if ( menu )
+	    menu->menuAction()->setVisible( visible );
```

Comment:
the indentation is inconsistent, please keep 4 spaces per level

and consider using EditorConfig to let the editor take care of it https://editorconfig.org/